### PR TITLE
Report metainfo endpoint

### LINKED ISSUE #715: Return "stored_at" timestamp in all endpoints that return results
Return "stored_at" timestamp in all endpoints that return results

### PR DESCRIPTION
**Pull request by tisnik**
# Description

Report metainfo endpoint:

```
{
  "metainfo": {
    "count": -1,
    "last_checked_at": "2020-02-02T00:00:00Z",
    "stored_at": "2020-01-01T00:00:00Z"
  },
  "status": "ok"
}
```

Fixes #715

## Type of change

- New feature (non-breaking change which adds functionality)
- Unit tests (no changes in the code)

## Testing steps

Done on CI or can be checked locally, of course.

## Checklist
* [ ] `make before_commit` passes
* [ ] updated documentation wherever necessary
* [ ] added or modified tests if necessary
* [ ] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

**Review by dpensi — APPROVED** [review]

File: server/server.go [hunk 1]
```diff
@@ -505,6 +505,59 @@ func (server HTTPServer) readAggregatorReportForClusterID(
...
 
+// readAggregatorReportMetainfoForClusterID reads report metainfo from Aggregator,
+// handles errors by sending corresponding message to the user.
+// Returns report and bool value set to true if there was no errors
+func (server HTTPServer) readAggregatorReportMetainfoForClusterID(
+	orgID ctypes.OrgID, clusterID ctypes.ClusterName, userID ctypes.UserID, writer http.ResponseWriter,
+) (*ctypes.ReportResponseMetainfo, bool) {
+	aggregatorURL := httputils.MakeURLToEndpoint(
+		server.ServicesConfig.AggregatorBaseEndpoint,
+		"organizations/{org_id}/clusters/{cluster}/users/{user_id}/report/info", //ira_server.ReportMetainfoEndpoint,
```
Comment (Bee-lee) [inline server/server.go:516]: hardcoded URL

Comment (tisnik) [reply server/server.go:516]: ah yup, the aggregator were not updated, I'll fix it soon

Comment (tisnik) [reply server/server.go]: fixed

**Review by Bee-lee — APPROVED** [review]
LGTM

**Pull request merged**